xdsl/dialects/riscv.py

```diff
@@ -3272,6 +3272,50 @@ def assembly_line(self) -> str | None:
         )
 
 
```

Comment:
```suggestion
# region RV32F: 9 “D” Standard Extension for Double-Precision Floating-Point, Version 2.0
```
This is another extension, so probably best to specify this even if this is only the start of the implementation.